blogposts/2020/3.18/announcing-sourcegraph-3.18.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Sourcegraph 3.18: Search across multiple revisions and non-master branches, custom search pages, campaigns now support GitLab, new C++ precise code intelligence"
+author: Laureen Hudson
+publishDate: 2020-MM-20T10:00-07:00
+tags: [blog]
+slug: sourcegraph-3.18
+heroImage: /blog/3.18-release-blog-img.jpg
+published: true
+---
+
+We’re improving Sourcegraph universal code search continuously, to help you solve the Big Code problems you face, all day every day. Here's how we're doing it.
+
+-   [Indexed non-master branches](https://about.sourcegraph.com/blog/indexed-non-master-branches)
+
+    Often, you need to search code that isn’t in your default branch, like long-lived release branches or important tags. In Sourcegraph 3.18, we now use an index on the default branch to speed up search operations, with the added capability to index branches other than the default branch.
+
+-   [Search across multiple revisions](https://about.sourcegraph.com/blog/search-multiple-revisions)
+
+    You can now search across multiple revisions of the same repository by listing multiple branch names (or other revision specifications) separated by `:` in your query. This helps you understand the differences between code at different branches (especially for release branches that have diverged).
+
+-   [Small but mighty new features](https://about.sourcegraph.com/blog/small-and-mighty-features)
+
+    Sourcegraph 3.18 includes several small features that have a big impact. Based on lots of feedback, we added support for campaigns in GitLab. We’ve also improved privacy feedback when adding repositories, and created custom search pages for Sourcegraph Cloud.
```

Comment:
The first sentence is a restatement of the "Small but mighty new features" title. I think the first sentence can be cut.

Reply:
done